www/scripts/codecheckerviewer/ListOfBugs.js

```diff
@@ -248,6 +265,7 @@ function (declare, dom, Deferred, ObjectStore, Store, QueryResults, topic,
         { name : 'Message', field : 'checkerMsg', width : '100%', formatter : checkerMessageFormatter },
         { name : 'Checker name', field : 'checkerId', width : '50%', formatter: checkerNameFormatter },
         { name : 'Severity', field : 'severity', cellClasses : 'severity', width : '15%', formatter : severityFormatter },
+        { name : 'Difficulty', field : 'bugPathLength', cellClasses : 'bug-path-length', width : '15%', formatter : bugPathLengthFormatter },
```

Comment:
Name it simply to Length or Bug Path length as it correctly expresses what the number means. "Difficulty" is not expressive enough